test/analytics/tests/instrumented_events/from_the_browser/loaded_dashboard.ts

```diff
@@ -8,9 +8,10 @@
 
 import { GetEventsOptions } from '@kbn/analytics-ftr-helpers-plugin/common/types';
 import expect from '@kbn/expect';
-import { DASHBOARD_LOADED_EVENT } from '@kbn/dashboard-plugin/public/events';
 import { FtrProviderContext } from '../../../services';
 
+const DASHBOARD_LOADED_EVENT = 'dashboard_loaded';
```

Comment:
Importing it was causing type errors because we are not supposed to import from nested folders. But also, I'd rather validate the expected behaviour instead of relying on a variable that may change (hence breaking the contract).